### ISSUE TITLE
Are be_alientaxa_info.csv file and the content of folder occurrenceCube still needed?

### ISSUE THREAD

**Issue by soriadelva** [inbo/aspbo#116]
@mvarewyck is the content of the occurrenceCube folder in ./data/output/UAT_processing/ still needed now that we have the following geopackage files in the UAT_processing/grid/folder? :

- https://github.com/inbo/aspbo/blob/adef403e84d5a2b0ddf4bfec5e7d7c1535d153d4/data/output/UAT_processing/grid/utm10_bel_with_regions.gpkg

- https://github.com/inbo/aspbo/blob/adef403e84d5a2b0ddf4bfec5e7d7c1535d153d4/data/output/UAT_processing/grid/utm1_bel_with_regions.gpkg

In addition, do we still need the be_alientaxa_info.csv file in the  ./data/output/UAT_processing/ folder?

**Comment by mvarewyck** [comment]
> @mvarewyck is the content of the occurrenceCube folder in ./data/output/UAT_processing/ still needed

No, it is redundant. In addition, these lines can be removed (otherwise pipeline might fail)
* https://github.com/inbo/aspbo/blob/uat/src/upload_processing_to_UAT.R#L95
* https://github.com/inbo/aspbo/blob/uat/src/upload_processing_to_UAT.R#L95

> In addition, do we still need the be_alientaxa_info.csv file in the ./data/output/UAT_processing/ folder?

Yes, that one we still use to create a file keys.csv which allows to match gbifKey to taxonKey in the app.
https://github.com/inbo/aspbo/blob/uat/src/upload_processing_to_UAT.R#L66
